src/main/java/com/theironyard/invitator/Main.java

```diff
@@ -0,0 +1,131 @@
+package com.theironyard.invitator;
+
+import spark.ModelAndView;
+import spark.Session;
+import spark.Spark;
+import spark.template.mustache.MustacheTemplateEngine;
+
+import java.util.*;
+
+/**
+ * Created by melmo on 12/22/16.
+ */
+public class Main {
+    static String SESSION_USERID = "username";
+//    static User user; // ??? Store user or get every time from db ???
+
+    public static void main(String[] args) {
+        Spark.staticFileLocation("/static");
+        Spark.init();
+
+        Spark.get("/", ((request, response) -> {
+            Map m = new HashMap();
+            Session session = request.session();
+            Integer currentUser = session.attribute(SESSION_USERID);
+
+            if (currentUser==null){
+                return new ModelAndView(m, "login.html");
+            }
+            else {
+                User user = User.loadUser(currentUser);
+                m.put("user", user);
+                m.put("events", user.getEvents().values());
+                return new ModelAndView(m, "user_events.html");
+            }
+        }), new MustacheTemplateEngine());
+
+        Spark.post("/login", ((request, response) -> {
+            String username = request.queryParams("username");
+            String password = request.queryParams("password");
+
+            if (username.isEmpty() || password.isEmpty()){
+                response.redirect("/");
+                return "";
+            }
+
+            User user = User.loadUser(username);
+
+            if (user==null){
```

Comment:
Watch the spaces here.  You should space out the variables and the `==`.  So better if it is `if(user == null)`